### PR TITLE
sources

### PR DESCRIPTION
**Pull request by SammySteiner**

### CODE REVIEW

**Comment by SammySteiner** [comment]
[Refactor citations (large)](https://trello.com/c/WXzh54Qe/64-refactor-citations-large)

File: gatsby-config.js [hunk 1]
```diff
@@ -96,6 +96,13 @@ module.exports = {
             })
           );
 
+          const sources = {
+            label: "Sources",
+            name: "sources",
+            widget: "list",
+            field: {label: "Source", name: "source", widget: "markdown"}
```
Comment (vonbearshark) [inline gatsby-config.js:103]: Are your precommits running? Should be automatically formatting this..

File: src/pages/state/[state].content.yml [hunk 2]
```diff
@@ -21,6 +21,9 @@ headings:
     heading: Donors Needed
   investigation:
     heading: Under Congressional Investigation
+sources:
+  - This is a test for the source list on state pages.
+  - This is a second test source for the state page. [^2]
```
Comment (vonbearshark) [inline src/pages/state/[state].content.yml:26]: `rm` reminder comment

File: src/pages/index.content.yml [hunk 3]
```diff
@@ -11,6 +11,14 @@ quote:
...
   attribution: New York Times editorial board
   image: ../images/quotes/mainPage.png
+sources:
+  - This is a test main page source 1.
+  - this is a test main page source 2.
+  - This is a very long test source on the main page 3.  This is a very long
+    test source on the main page 3.  This is a very long test source on the main
+    page 3.  This is a very long test source on the main page 3.  This is a very
+    long test source on the main page 3.  This is a very long test source on the
+    main page 3.
```
Comment (vonbearshark) [inline src/pages/index.content.yml:21]: `rm` reminder comment

File: src/pages/equity.content.yml [hunk 4]
```diff
@@ -79,3 +79,5 @@ smallQuotes:
       system as a matter of life and death.
     attribution: Congresswoman Ayanna Pressley
 tweet: "1388204146969427971"
+sources:
+  - Source for the equity page.
```
Comment (vonbearshark) [inline src/pages/equity.content.yml:83]: `rm` reminder comment

File: src/components/layout/layout.js [hunk 5]
```diff
@@ -43,6 +44,20 @@ export default function Layout({ crumbLabel, children }) {
         </Breadcrumb>
       ) : null}
       {children}
+      {sources?.length ? (
+          <Row className={styles.sources}>
+            <h3>Sources</h3>
+            <ol>
+              {Object.values(sources)
+                .sort((a, b) => a.index - b.index)
```
Comment (vonbearshark) [inline src/components/layout/layout.js:52]: Can `rm` this `.sort`

**Review by vonbearshark — APPROVED** [review]
lgtm!

**Review by vonbearshark — CHANGES_REQUESTED** [review]
Thinking through this earlier and I think this PR should include the changes to remove the citations from the table heading in the CMS and potentially refactor the table heading CMS model? Maybe an object with keys is not the best shape? (Maybe it is, though, so just removing the citations in that case)

File: src/components/layout/layout.js [hunk 6]
```diff
@@ -43,6 +44,20 @@ export default function Layout({ crumbLabel, children }) {
         </Breadcrumb>
       ) : null}
       {children}
+      {sources?.length ? (
+          <Row className={styles.sources}>
```
Comment (vonbearshark) [inline src/components/layout/layout.js:48]: Should we give this an ID for anchor linking?

Comment (SammySteiner) [reply src/components/layout/layout.js:48]: I have an id for each li element, that way we can link directly to the specific footnote. They are ordered and display in the same order as in the cms.

File: src/components/quoteWithImage/quoteWithImage.js [hunk 7]
```diff
@@ -13,7 +13,7 @@ export default function QuoteWithImage({
   const [{ quoteImagesByPath }] = useQuoteImages();
 
   return (
-    <Row>
+    <>
```
Comment (vonbearshark) [inline src/components/quoteWithImage/quoteWithImage.js:16]: nit: necessary if it's already wrapped in `BgImage`?

**Review by vonbearshark — APPROVED** [review]
re: the existing table headings in the CMS, the titles do not have to be nested object widgets. But it's not must-fix

**Pull request merged**